Sources/interoperability-tests/InteroperabilityTestsExecutable.swift

```diff
@@ -60,4 +60,98 @@ struct InteroperabilityTestsExecutable: AsyncParsableCommand {
       }
     }
   }
+
+  struct RunTests: AsyncParsableCommand {
+    static var configuration = CommandConfiguration(
+      abstract: """
+          Run gRPC interoperability tests using a gRPC Swift client.
+          You can specify a test name as an argument to run a single test.
+          If no test name is given, all interoperability tests will be run.
+        """
+    )
+
+    @Option(help: "The host the server is running on")
+    var host: String
+
+    @Option(help: "The port to connect to")
+    var port: Int
+
+    @Argument(help: "The name of the test to run. If absent, all tests will be run.")
+    var testName: String?
```

Comment:
Sorry, I meant that we can specify _multiple_ arguments, i.e. `var testNames: [String]`

Reply:
Yes, if it's empty then run everything. If you don't want to run any tests then don't call the CLI 🙂.